docs/docs/references/user-attributes.mdx

```diff
@@ -38,10 +38,16 @@ frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media
 User attributes can be created by navigating to **Organization Settings** > **User Attributes** and clicking on the 
 **Add attribute** button. This will create a new user attribute but it will not be assigned to any user.
 
-### Assigning user attributes to users
+### Assigning user attributes to users and groups
 
-User attributes can be assigned to users by navigating to **Organization Settings** > **User Attributes** and 
-clicking on the user attribute you'd like to assign. Select a user by email address and set their value.
+User attributes can be assigned to users or [groups](/references/groups.mdx) by navigating to **Organization Settings** > **User Attributes** and 
+clicking on the user attribute you'd like to assign. Select a user by email address, or a group by group name and set their value.
+
+:::info
+
+If a user is assigned a value for an attribute and they also belong to a group that's been assigned a different value for the same attribute, the value for the user will take precedence over the value for the group.
```

Comment:
We should be specific about when we only check 1 value vs all values. 

Requirements from [use group attributes in column and row access#8500](https://github.com/lightdash/lightdash/issues/8500)

> Column security - if the attribute matches a user or any group attribute

eg: user has attribute value 'apple' and is assigned to 2 groups with the attribute values 'orange' and 'kiwi'

column yml
```
    meta: 
      dimension: 
        required_attributes: 
          fruit: "kiwi" 
```

In this case, it will show the column since 'kiwi' is present in the user attributes ['apple', 'orange', 'kiwi']

> Row security - User attributes take precedence over group attributes. Group attribute behaviour not defined, if multiple groups reference the same value

model yml
```
models:
  - name: my_model
    meta:
      sql_filter: ${TABLE}.fruit = ${lightdash.attributes.fruit}
```  

Scenario 1: 
attribute default value: 'banana'
user value: 'apple'
groups values: 'orange' and 'kiwi'

`lightdash.attributes.fruit` will be 'apple' // User attributes take precedence over group attributes

Scenario 2: 
attribute default value: 'banana'
user value: (no value)
groups values: 'orange' and 'kiwi'

`lightdash.attributes.fruit` will be 'orange' or 'kiwi' // Group attribute order is not defined 

Scenario 3: 
attribute default value: 'banana'
user value: (no value)
groups values: (no value)

`lightdash.attributes.fruit` will be 'banana' // use default value 

Note: There is an open ticket so developers can have access to all the values.  [Add reference to all attribute values in row level filtering#8536](https://github.com/lightdash/lightdash/issues/8536)